.gitignore

```
@@ -13,7 +13,7 @@ built/*
 build
 node_modules
 .node_modules/
-daml.js/*
+ui/daml.js
```

Wouldn't this do and be a bit more robust?
```suggestion
daml.js
```